torch/csrc/jit/runtime/symbolic_shape_registry.cpp

```diff
@@ -105,6 +106,9 @@ static const OperatorMap<std::string>& get_schema_to_function_graph() {
       {"aten::cat(Tensor[] tensors, int dim=0) -> Tensor", "cat"},
       {"aten::permute(Tensor(a) self, int[] dims) -> Tensor(a)", "permute"},
       {"aten::view(Tensor(a) self, int[] size) -> Tensor(a)", "view"},
+      {"aten::reshape(Tensor(a) self, int[] shape) -> Tensor(a)", "view"},
```

Comment:
I don't know to what extend we'll need these.... (Elias pointed to me earlier that some of these ops are handled in a different file, but we can clean it up in the next upstream PR)

Reply:
I should remove/update the aten::gelu_backward entry.